apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/Labels.cs

```diff
@@ -42,6 +53,20 @@ internal static class Labels
         ///<summary>The label to denote a stack trace.</summary> 
         internal const string StackTrace = "trace.cloud.google.com/stacktrace";
 
+        ///<summary>The label to denote an agent.</summary> 
+        internal const string Agent = " /agent";
+
+        /// <summary>
+        /// Gets a map with the label for the agent which contains the agent's name and version.
+        /// </summary>
+        internal static Dictionary<string, string> AgentLabel()
+        {
+            return new Dictionary<string, string>()
```

Comment:
Stylistic nit: for object/collection initializers where you're calling a parameterless constructor, I'd typically omit the `()`.